content/blog/2018-05-12-nsm-skydive-integration.md

```diff
@@ -0,0 +1,41 @@
+# Using skydive to visualize nsm concepts
+
+It is always tricky for infrastructure projects, and especially networking infrasctructure projects, to explain their concepts, and what the project does effectively.
+One always better understand concepts he can visualize over those explained with hundreds of words.
+It is even harder for networking infractructure projects that introduce new way of interconnecting components across the underlying infrastructure. That's what the network service does.
+
+To tackle down this situation, we propose to use Skydive.
```

Comment:
I would suggest "we use Skydive" (an actual fact) rather that "propose to use Skydive" (a proposal).  You guys have done lots of super good work... own it :)